README.md

```diff
@@ -34,13 +34,24 @@ When the connector switches to CDC mode, it starts streaming changes from the
 
+### Unsafe snapshot
+
+By default, the connector will error out if it finds a table that has no primary key and no specified sorting column specified, as we can't guarantee that the snapshot will be consistent. Table changes during the snapshot will be however captured by CDC mode.
+
+As of writing, the unsafe snapshot is implemented using batches with `LIMIT` and `OFFSET`, so expect it to be slow for large tables. You can optimize the snapshot by specifying a [sorting column](#configuration).
+The position of the table is currently not recorded, so the unsafe snapshot will restart from zero every time.
+
 ### Configuration
 
-| name                                     | description                                                                                                                                             | required | default value | example                                                         |
-| ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------------- | --------------------------------------------------------------- |
-| `dsn`                                    | [The data source name](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name) for the MySQL database.                          | true     |               | <user>:<password>@tcp(127.0.0.1:3306)/<db>?parseTime=true       |
-| `tables`                                 | The list of tables to pull data from                                                                                                                    | true     |               | users,posts,admins                                              |
-| `tableConfig.<table name>.sortingColumn` | The custom column to use to sort the rows during the snapshot. Use this if there are any tables which don't have a proper autoincrementing primary key. |          |               | `tableConfig.users.sortingColumn` as the key, `id` as the value |
+| name                                        | description                                                                                                                                             | required | default | example                                                         |
+| ------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- | ------- | --------------------------------------------------------------- |
+| `dsn`                                       | [The data source name](https://github.com/go-sql-driver/mysql?tab=readme-ov-file#dsn-data-source-name) for the MySQL database.                          | true     |         | <user>:<password>@tcp(127.0.0.1:3306)/<db>?parseTime=true       |
```

Comment:
`<user>:<password>` are not shown in the parsed Markdown file.

Reply:
Should be changed from the previous pr now.